.github/workflows/publish.yml

```diff
@@ -64,16 +57,140 @@ jobs:
+        with:
+          name: packages-linux
+          path: dist
+
+  create-packages-darwin:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          submodules: recursive
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Setup Mage
+        run: |
+          brew install mage
+
+      - name: Build OpenSSL 3.0
+        run: |
+          wget https://github.com/openssl/openssl/releases/download/openssl-3.0.8/openssl-3.0.8.tar.gz
+          tar -xvf openssl-3.0.8.tar.gz
+          cd openssl-3.0.8/
+          ./Configure --prefix=${GITHUB_WORKSPACE}/openssl no-shared
+          make && make install
+
+      - name: Setup GoReleaser
+        run: |
+          brew install goreleaser
+
+      - name: Set GoReleaser flags
+        id: set-goreleaser-flags
+        run: |
+          if ${{ startsWith(github.ref, 'refs/tags') }} ; then
+            echo "::set-output name=GORELEASER_FLAGS::--rm-dist --skip-validate"
+          else
+            echo "::set-output name=GORELEASER_FLAGS::--rm-dist --snapshot"
+          fi
+
+      - name: Build packages
+        env:
+          CGO_LDFLAGS: "-L${{ env.GITHUB_WORKSPACE }}/openssl/lib64"
+          CGO_CFLAGS: "-I${{ env.GITHUB_WORKSPACE }}/openssl/include"
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          goreleaser release ${{ steps.set-goreleaser-flags.outputs.GORELEASER_FLAGS }} --skip-publish --config .goreleaser_macOs.yml
 
       - name: Upload packages artifacts
         uses: actions/upload-artifact@v2
         with:
-          name: packages
+          name: packages-darwin
           path: dist
 
-  publish-s3:
-    needs: create-packages
+  draft-release:
+    needs: [create-packages-linux, create-packages-darwin]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          submodules: recursive
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      # These scripts will be picked up while building packages with goreleaser.
+      - name: Generate ZSH and Bash completion scripts
+        run: |
+          mage build
+          ./tt completion bash > tt-completion.bash
+          ./tt completion zsh > tt-completion.zsh
+
+      - name: Make directories
+        run: |
+          mkdir -p ./dist-prebuilt/packages-linux
+          mkdir -p ./dist-prebuilt/packages-darwin
+
+      - name: Download linux binaries
+        uses: actions/download-artifact@v3
+        with:
+          name: packages-linux
+          path: ./dist-prebuilt/packages-linux
+
+      - name: Download darwin binaries
+        uses: actions/download-artifact@v3
+        with:
+          name: packages-darwin
+          path: ./dist-prebuilt/packages-darwin
+
+      - name: Remove not-needed files
```

Comment:
How did you get the list of files that need to be removed?

Reply:
I have looked what files goreleaser produce, than removed not-needed. I don't think that hardcode will cause troubles.